app-i18n/mozc/mozc-2.23.2785.102.recipe

```diff
@@ -57,7 +57,8 @@ BUILD()
 
 	GYP_DEFINES="document_dir=$dataDir/mozc \
 		zinnia_model_file=$dataDir/mozc/zinnia/handwriting-ja.model \
-		zinnia_tomoe_dir=../../../../../sources-2/zinnia-tomoe-0.6.0-20080911" \
+		zinnia_tomoe_dir=../../../../../sources-2/zinnia-tomoe-0.6.0-20080911 \
```

Comment:
You can (probably) use `$sourceDir2`:
```diff
-		zinnia_tomoe_dir=../../../../../sources-2/zinnia-tomoe-0.6.0-20080911 \ 
+		zinnia_tomoe_dir=$sourceDir2/zinnia-tomoe-0.6.0-20080911 \ 
```

Reply:
Hmm, I just noticed that `SOURCE_URI_2` can be improved a little:
```diff
-SOURCE_URI_2="https://jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/0.6.0-20080911/zinnia-tomoe-0.6.0-20080911.tar.bz2"
+tomoeVersion="0.6.0-20080911"
+SOURCE_URI_2="https://downloads.sourceforge.net/zinnia/zinnia-tomoe-$tomoeVersion.tar.bz2"
```
This means you could even use the following change, instead of what I suggested in the previous comment:
```diff
-		zinnia_tomoe_dir=../../../../../sources-2/zinnia-tomoe-0.6.0-20080911 \ 
+		zinnia_tomoe_dir=$sourceDir2/zinnia-tomoe-$tomoeVersion \ 
```

Reply:
Thanks. I'll try to make the recipe better according to your suggestions.